### PR TITLE
formatting unification

### PR DESCRIPTION
**Pull request by sphinxc0re**
child of #106 

- Tests now check for proper formatting
- added `docker/run-fmt` for running the formatting task inside docker

### CODE REVIEW

File: docker/run-fmt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker run -v `pwd`:/holochain --rm -p 3141:3141 -it holochain/rust-ubuntu:latest cargo fmt
```
Comment (thedavidmeister) [inline docker/run-fmt]: @sphinxc0re FYI it's `holochain/rust-ubuntu:develop` as we don't have a `:latest` tag atm, i'll update and push

**Comment by thedavidmeister** [comment]
this is much faster than running a full test build!

`. docker/run-fmt` to run it

**Comment by sphinxc0re** [comment]
I'll update the README as well

**Comment by sphinxc0re** [comment]
@thedavidmeister Could you take another look at this?

**Comment by thedavidmeister** [comment]
@sphinxc0re yeah it's all good, thanks for sweeping the readme

**Pull request merged**